openvdb/util/PagedArray.h

```diff
@@ -200,9 +201,10 @@ class PagedArray
 
 public:
     using ValueType = ValueT;
+    using Ptr       = SharedPtr<PagedArray>;
 
     /// @brief Default constructor
-    PagedArray() { mSize = 0; }
+    PagedArray() = default;
```

Comment:
Revert this change; `tbb::atomic` needs to be initialized by assignment.  (See PR #386.)

Reply:
done